src/components/grid-aware/OneParagraphBlock/OneParagraphBlock.js

```diff
@@ -0,0 +1,54 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import Button from "../../inline/Button";
+
+import s from "./OneParagraphBlock.module.css";
+
+const OneParagraphBlock = ({ title, jobs, ctaButtons }) => {
+  const GridAreaLeft = () => (
+    <div className={s.gridAreaLeft}>
+      <div className={s.title}>{title}</div>
+    </div>
+  );
+
+  const GridAreaRight = () => (
+    <div className={s.gridAreaRight}>
+      <div className={s.jobsWrapper}>
+        <div className={s.jobs}>{jobs}</div>
+        <div className={s.ctaButtonRow}>
+          {ctaButtons.map((button) => (
+            <div className={s.ctaButtonItem} key={button.text}>
+              <Button
+                text={button.text}
+                internalLink={button.internalLink}
+                externalLink={button.externalLink}
+                onClick={button.onClick}
+              />
+            </div>
+          ))}
+        </div>
+      </div>
+    </div>
+  );
+
+  return (
+    <div className={s.bleedWrapper}>
+      <div className={s.bleedBackground} />
+      <div className={s.bleedMainContent}>
+        <section className={s.gridParent}>
+          <GridAreaLeft />
+          <GridAreaRight />
+        </section>
+      </div>
+    </div>
+  );
+};
+
+OneParagraphBlock.propTypes = {
+  title: PropTypes.string.isRequired,
+  jobs: PropTypes.node.isRequired,
```

Comment:
Since this component has the generic name of "one-paragraph block", we should expect to use it for a number of other use cases as well, so we probably shouldn't name this `jobs`. How about `description`, which would make it match the `title` and `description` props of the 2- and 3-paragraph block components?

We should also rename the associated CSS classes.